### PR TITLE
added README.md for NYC fares and Electricity projects, fixed 5th notebook in Electricity. 

### PR DESCRIPTION
**Pull request by rktraz**

### CODE REVIEW

**Comment by Maxxx-zh** [comment]
⚠️ Problem with **td_transformation_functions** retrieval.

Знімок екрана 2022-10-06 о 9 21 17 пп
Знімок екрана 2022-10-06 о 9 21 26 пп

File: advanced_tutorials/electricity/README.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+Engineered features are typically also stored in feature groups in the feature store.
+
+
+### Preparing Data for Model Training
+
+Hopsworks makes it easy to create datasets for model training.
+
+In the Hopsworks framework, training datasets are immutable in the sense that, in contrast to feature groups, you cannot append data to them after they have been created. However, you *can* have different *versions* of the same training dataset.
+
```
Comment (davitbzh) [inline advanced_tutorials/electricity/README.md:84]: Write about feature views and point to docs. TD in created from FV not FG

File: advanced_tutorials/nyc_taxi_fares/1_backfill_feature_groups.ipynb [hunk 2]
```diff
@@ -11,9 +11,9 @@
     "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/logicalclocks/hopsworks-tutorials/blob/master/advanced_tutorials/nyc_taxi_fares/1_backfill_feature_groups.ipynb)\n",
     "\n",
     "## 🗒️ This notebook is divided into 3 sections:\n",
-    "1. Load some data from files.\n",
-    "2. Connect to the Hopsworks Feature Store.\n",
-    "3. Create feature groups and insert them into the Feature Store.\n",
+    "1. **Load some data from files**,\n",
```
Comment (davitbzh) [inline advanced_tutorials/nyc_taxi_fares/1_backfill_feature_groups.ipynb:14]: avoid word "some". Write just "load data from source files"

Comment (davitbzh) [inline advanced_tutorials/nyc_taxi_fares/1_backfill_feature_groups.ipynb:14]: avoid word "some". Write just "load data from source files"

File: advanced_tutorials/nyc_taxi_fares/2_feature_pipeline.ipynb [hunk 3]
```diff
@@ -10,8 +10,8 @@
     "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/logicalclocks/hopsworks-tutorials/blob/master/advanced_tutorials/nyc_taxi_fares/2_feature_pipeline.ipynb)\n",
     "\n",
     "## 🗒️ This notebook is divided into 2 sections:\n",
-    "1. Data generation.\n",
-    "2. Insertion new data into Feature Store."
+    "1. **Data generation**,\n",
+    "2. **Insertion new data into Feature Store**."
```
Comment (davitbzh) [inline advanced_tutorials/nyc_taxi_fares/2_feature_pipeline.ipynb:14]: Insert new data into the Feature Store

Comment (davitbzh) [inline advanced_tutorials/nyc_taxi_fares/2_feature_pipeline.ipynb:14]: Insert new data into the Feature Store

**Review by davitbzh — CHANGES_REQUESTED** [review]

**Comment by davitbzh** [comment]
Also please remove cached files

**Comment by rktraz** [comment]
wait a lil bit, I want to improve streamlit + `get_train_test_split` returns in new style

File: advanced_tutorials/nyc_taxi_fares/README.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  <b>Streamlit</b> app will be created so you would be able to try your model interactively.
+
+   This is a <b>batch use case</b>, it will give you a high-level view of how to use our python APIs and the UI to navigate the feature groups.
+ </span>
+
+## **🗒️ This whole tutorial is divided into 5 parts:**
+1. How to backfill Features to the Feature Store,
+2. How to create a feature pipeline,
+3. How to create Feature views & Training Datasets,
+4. How to train a model and upload it to the Model Registry,
```
Comment (davitbzh) [inline advanced_tutorials/nyc_taxi_fares/README.md:21]: remove "How to "

File: advanced_tutorials/nyc_taxi_fares/README.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+5. Streamlit app.
```
Comment (davitbzh) [inline advanced_tutorials/nyc_taxi_fares/README.md:22]: Deploy streamlit app

**Review by davitbzh — CHANGES_REQUESTED** [review]

File: advanced_tutorials/electricity/1_backfill_feature_groups.ipynb [hunk 6]
```diff
@@ -18,9 +18,9 @@
     "The objective of this tutorial is to demonstrate how to work with the **Hopworks Feature Store**  for batch data with a goal of training and deploying a model that can predict electricity prices in the future.\n",
     "\n",
     "## 🗒️ This notebook is divided in 3 sections:\n",
-    "1. Loading the data and feature engineeing,\n",
-    "2. Connect to the Hopsworks feature store,\n",
-    "3. Create feature groups and upload them to the feature store."
+    "1. **Loading the data and feature engineeing**,\n",
```
[the remark this answers is not in the document]
Comment (rktraz) [reply advanced_tutorials/electricity/1_backfill_feature_groups.ipynb:21]: Davit, why don't you like such constructions like _"Loading, modeling..."_?

It s used in many tutorials on the internet
And in the sentence _"Loading the data and feature engineering"_  there is a _"feature engineering"_ part that is strange to change to "engineer features"

File: advanced_tutorials/electricity/README.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+   This is a <b>batch use case</b>, it will give you a high-level view of how to use our python APIs and the UI to navigate the feature groups.
+ </span>
+
+## **🗒️ This whole tutorial is divided into 5 parts:**
+1. How to backfill Features to the Feature Store,
+2. How to create a feature pipeline,
+3. How to create Feature views & Training Datasets,
+4. How to train a model and upload it to the Model Registry,
+5. How to fetch model from Model Registry and make batch predictions.
+
```
[the remark this answers is not in the document]
Comment (rktraz) [reply advanced_tutorials/electricity/README.md:22]: I think it is comfortable when we have fundamental questions that our project is divided into, and in each part, we answer one of these questions

So should I write
`
...
3. Create Feature views & Training Datasets,
4. Train a model and upload it to the Model Registry,
5. Fetch the model from Model Registry and make batch predictions...
`

?

**Review by davitbzh — APPROVED** [review]

**Pull request merged**